<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>525</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>525</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>525</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget {border-image: url(:/image/accesso.jpg);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="LineEditUsernameForm">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>216</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-radius: 15px;
padding: 0 8px;
selection-background-color: darkgray;
font-size: 16px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEditPasswordForm">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>263</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-radius: 15px;
padding: 0 8px;
selection-background-color: darkgray;
font-size: 16px;
</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoginButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>310</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#LoginButton{color: rgb(255, 255, 255); 
background-color: rgb(20, 115, 255); 
font-size: 16px; 
border-radius: 20px; 
border: 3px solid rgb(28, 110, 164);
}
#LoginButton:hover{
background-color: rgb(28, 110, 164);
}
#LoginButton:pressed{
background-color: rgb(20, 78, 117);
}</string>
    </property>
    <property name="text">
     <string>Accedi</string>
    </property>
   </widget>
   <widget class="QLabel" name="version">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #EAFF00;</string>
    </property>
    <property name="text">
     <string>QuestoTestoVieneSostituitoDaVersione</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ForgotPasswordButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>360</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#ForgotPasswordButton {
color: #505050;
background-color: Transparent;
background-repeat:no-repeat;
}

#ForgotPasswordButton:hover {
color: #505050;
background-color: Transparent;
background-repeat:no-repeat;
text-decoration: underline;
}</string>
    </property>
    <property name="text">
     <string>Password dimenticata?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RegistratiButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>480</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#RegistratiButton {
color: #0033CC;
background-color: Transparent;
background-repeat:no-repeat;
}

#RegistratiButton:hover {
color: #0040FF;
text-decoration: underline;
}</string>
    </property>
    <property name="text">
     <string>Registrati</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>480</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sei nuovo?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #EAFF00;</string>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_connect">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>350</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect to server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_disconnect">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>350</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect from server</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_log">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>380</y>
      <width>291</width>
      <height>87</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>0</y>
      <width>29</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: Transparent;
background-repeat:no-repeat;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="LoginAdmin">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>400</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#LoginAdmin{color: rgb(255, 255, 255); 
background-color: rgb(255, 0, 0); 
font-size: 16px; 
border-radius: 20px; 
border: 3px solid #B30000;
}
#LoginAdmin:hover{
background-color: #B30000;
}
#LoginAdmin:pressed{
background-color: #800000;
}</string>
    </property>
    <property name="text">
     <string>Accedi (Admin Mode)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
